spec/system/signup/signup_spec.rb

```diff
@@ -9,10 +9,11 @@
 
     test_email = gmail.account_email.gsub(/@/, "+#{Time.now.to_i}@")
 
-    if ENV["STAGING"] 
+    if (ENV["STAGING"] = 1)
       emailaddress="signup@staging.wifi.service.gov.uk"
     else
       emailaddress="signup@wifi.service.gov.uk"
```

Comment:
Suspect rubocop fixes didn't make the commit? It won't like the lack of spaces around `=`